src/controllers/bookController.ts

```diff
@@ -1,12 +1,15 @@
 import { Router, Request, Response } from 'express';
+import Book from '../book';
+import { Connection as TediousConnection, Request as TediousRequest } from 'tedious';
+import { dbConfig } from '../dbConfig';
 
 class BookController {
     router: Router;
 
     constructor() {
         this.router = Router();
+        this.router.get('/catalogue', this.getAllBooks.bind(this));
```

Comment:
I'd argue that this route should just be `GET /` rather than `GET /catalogue`. We're already binding this controller to the base route `/books`, and we're just getting a list of books here, so the final request ends up being `GET /books`, which is more semantic I think